### DISCUSSION TITLE
slider velocity isnt being saved properly

### DISCUSSION THREAD

**Discussion by Axilotl17** [ppy/osu#16537] [Q&A] [answered]
[I've recorded a video of the bug here](https://youtu.be/Nj7DUF_gI_k)
(sorry for low quality idk why that happened)

I've tried restarting my game, I've tried reinstalling, different maps too; the only solution seems to be to use stable.

text version of bug:
the slider velocity sometimes gets reset to 1, or to the slider velocity of preceding sliders, and when you try to fix it it doesn't save, and even sometimes spreads to other nearby sliders. and when i change the position and velocity of a slider, only the position get saved, not the velocity. 

I'm using arch Linux, nvidia gpu and drivers. running lazer 2022.118.0 (although this happens across versions)

**Comment by peppy** [comment, marked as answer]
We can't help unless you provide the beatmap, timestamps of the hitobjects you're changing, etc.

Restarting won't help. Reinstalling won't help. Also please provide logs so we can track what you did in the session that may affect this.

**Reply by Axilotl17** [reply]
this happens on all beatmaps, even when i edit previously uploaded ones i didnt make; but ill send it anyways. 

Im also unsure which logs you want so ill just zip the logs folder and send that
[logs.zip](https://github.com/ppy/osu/files/7909793/logs.zip)

as i said in the video i sent, this happens all over the map, on all maps, and it was totally fine yesterday, but is now broken, and it doesnt save when i fix it. this has been happening for weeks now, this isn't some new thing specific to this map.
the first instance of it happening for me is on 14:887, although literally in the video the first instance was at 17: something. so its variable.
[BLUE ENCOUNT - Survivor (Axilotl).zip](https://github.com/ppy/osu/files/7909798/BLUE.ENCOUNT.-.Survivor.Axilotl.zip)

**Reply by peppy** [reply]
~~Are you using the appimage?~~

Seems I can reproduce some of the time on the latest release, while never on the previous release. Will investigate.

**Reply by Axilotl17** [reply]
Sorry for late reply, I am using from AUR. thank you for the investigating x) much appreciated. this issue has been plaguing my mapping for awhile and mapping is one of my favorite things about osu. beautiful game you have made here. best of luck fixing it! l

**Reply by peppy** [reply]
As a quick fix, you can roll back to the previous release. I've confirmed it doesn't have the same issue.

**Reply by Axilotl17** [reply]
i thought it did, but i guess im mistaken.
